index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>portfolio mobile first</title>
+  </head>
+  <body>
+    <div class="container">
+      <nav>
+        <p id="name">Henry Ik</p>
+        <img src="images/Union.png" alt="menu" />
```

Comment:
- [ ] for semantic purposes, kindly put only the hamburger icon inside the nav tag. nice work